python/dask_cudf/dask_cudf/io/parquet.py

```diff
@@ -40,42 +40,82 @@ def read_metadata(*args, **kwargs):
 
         return (new_meta, stats, parts, index)
 
+    @classmethod
+    def multi_support(cls):
```

Comment:
Where is this being used ?

Reply:
It is used upstream (in Dask) to check if we support `pieces` being passed in a as a list in `read_partition`.  So, it is an "opt-in" mechanism.